crates/runtime/src/init/view.rs

```diff
@@ -83,34 +93,29 @@ impl Runtime {
 
     fn load_view(&self, view: &View, secrets: Arc<RwLock<Secrets>>) -> Result<()> {
         let df = Arc::clone(&self.df);
-        df.register_view(
-            view.name.clone(),
-            view.sql.clone(),
-            view.acceleration.clone(),
-            secrets,
-        )
-        .context(UnableToAttachViewSnafu)
-        .inspect_err(|_| {
-            self.status
-                .update_view(&view.name, status::ComponentStatus::Error);
-        })?;
+        df.register_view(view.clone(), secrets)
```

Comment:
The view clone is relatively expensive because the SQL that represent the view is stored directly. Can we store the SQL as an Arc<str>?

Reply:
Updated: updated View.sql from String to Arc<str>